litestack.gemspec

```diff
@@ -36,4 +36,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "hanami-router"
   spec.add_dependency "tilt"
   spec.add_dependency "erubi"
+  
+  spec.add_development_dependency "rake"
+  spec.add_development_dependency "railties"
```

Comment:
I was also running into this, but I'm wondering if we really need to add `railties` as a dependency. 

I think we would just need to update the `if` condition here:
https://github.com/oldmoe/litestack/blob/8a061b849f2e3dd137f398fca0fd99a8ea5451c0/lib/litestack.rb#L19

To something like `if defined(Rails::Railtie)`

Reply:
I went ahead and proposed that here: https://github.com/oldmoe/litestack/pull/20

Reply:
I like your solution for production, but am only adding railties as a dev dependency to make testing easier

Reply:
Oh, I see. That makes more sense